pom.xml

```diff
@@ -52,7 +52,7 @@
         <jackson.version>2.9.4</jackson.version>
         <jacoco-maven-plugin.version>0.7.9</jacoco-maven-plugin.version>
         <junit.version>4.12</junit.version>
-        <kafka.version>1.0.1-SNAPSHOT</kafka.version>
+        <kafka.version>1.0.2-SNAPSHOT</kafka.version>
```

Comment:
Need to update https://github.com/confluentinc/common-tools/blob/master/confluent/config/dev/versions.json#L26

Reply:
I can file a PR for that. I guess we shouldn't actually merge this until 4.0.1 goes out anyway, we're just going to be stuck w/ broken kafka-1.0 builds until that happens.